pkg/resource_lookup/resource_lookup.go

```
@@ -0,0 +1,43 @@
+package resource_lookup
```

golangci-lint (1.62.2)

[error] var-naming: don't use an underscore in package name

(revive)